encrypt-app.jsx

```diff
@@ -3,6 +3,8 @@ import React from 'react';
 /* eslint-disable no-unused-vars */
 import { render } from 'react-dom';
 import routes from './encrypt-main.jsx';
+import ga  from 'react-ga';
 import { Router, browserHistory } from 'react-router';
 
+ga.initialize(process.env.GA_TRACKING_ID);
```

Comment:
@alanmoo we have to put this on heroku config before we merge and deploy/promote to production, so that it will pick up the env once it deployed.